create-utils.sh

```diff
@@ -95,28 +95,28 @@ if [ "${build_dwarfs}" = "true" ]; then
     #
     # This can also be worked around by setting LC_ALL=C, but for now
     # let's revert the commit
-    cd dwarfs
+    cd dwarfs || exit 1
     git revert --no-commit aeeddaecab5d4648780b0e11dc03fca19e23409a
 
 	mkdir build
-	cd build
+	cd build || exit 1
 	cmake .. -DCMAKE_BUILD_TYPE=Release \
 			-DPREFER_SYSTEM_ZSTD=ON -DPREFER_SYSTEM_XXHASH=ON \
 			-DPREFER_SYSTEM_GTEST=ON -DPREFER_SYSTEM_LIBFMT=ON
 
-	make -j$(nproc)
+	make -j"$(nproc)"
 	make DESTDIR="${script_dir}"/build-utils/bin install
 
-	cd "${script_dir}"/build-utils
+	cd "${script_dir}"/build-utils || exit 1
 	mv bin/usr/local/sbin/dwarfs2 utils/dwarfs
 	mv bin/usr/local/sbin/dwarfs utils/dwarfs3
 	mv bin/usr/local/bin/mkdwarfs utils
 	mv bin/usr/local/bin/dwarfsextract utils
 fi
 
-libs_list="$(ldd utils/* | grep "=> /" | awk '{print $3}' | xargs)"
+libs_list=("$(ldd utils/* | awk '/=> \// {print $3}')")
 
-for i in ${libs_list}; do
+for i in "${libs_list[@]}"; do
```

Comment:
This does not work for some reason, seems like it keeps '\n' at the end of each element.